src/Symfony/Component/JsonCrawler/Exception/InvalidJsonException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonCrawler\Exception;
+
+class InvalidJsonException extends \RuntimeException
```

Comment:
Can you describe this exception, because if it is really about invalid json, it should extend `\InvalidArgumentException`, no?

Reply:
Indeed, fixed this part. Thanks 👍